ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetBulk.java

```diff
@@ -91,25 +100,45 @@ public GetBulkResult call() throws Exception {
             getter = new FileObjectGetter(this.outputPath);
         }
 
+        if (sync) {
+            if (this.prefix == null) {
+                LOG.info("Syncing all objects from " + this.bucketName);
+            }
+            else {
+                LOG.info("Syncing only those objects that start with " + this.prefix);
+            }
+            return new GetBulkResult(restoreSome(getter));
+        }
+
         if (this.prefix == null) {
             LOG.info("Getting all objects from " + this.bucketName);
             return new GetBulkResult(restoreAll(getter));
         }
-        else {
-            LOG.info("Getting only those objects that start with " + this.prefix);
-            return new GetBulkResult(restoreSome(getter));
-        }
+
+        LOG.info("Getting only those objects that start with " + this.prefix);
+        return new GetBulkResult(restoreSome(getter));
     }
 
     private String restoreSome(final Ds3ClientHelpers.ObjectChannelBuilder getter) throws IOException, SignatureException, XmlProcessingException, SSLSetupException {
         final Ds3ClientHelpers helper = getClientHelpers();
         final Iterable<Contents> contents = helper.listObjects(this.bucketName, this.prefix);
-        final Iterable<Ds3Object> objects = Iterables.transform(contents, new Function<Contents, Ds3Object>() {
-            @Override
-            public Ds3Object apply(final Contents input) {
-                return new Ds3Object(input.getKey(), input.getSize());
+
+        Iterable<Contents> filteredContents = null;
```

Comment:
This can be made final by having an else after `if (sync)` that assigns it to null.